py/server/tests/test_dbc.py

```diff
@@ -34,14 +34,57 @@ def test_read_adbc(self):
                 # instead of the actual size
                 self.assertEqual(table.size, 10)
 
-    @unittest.skipIf(platform.machine() != "x86_64", reason="connectorx not available on Linux/Arm64")
-    def test_read_sql(self):
-        postgres_url = "postgresql://test:test@postgres:5432/test"
+    @unittest.skipIf(platform.machine() != "x86_64", reason="connectorx not available on Linux/Arm64"
+                                                            "https://github.com/sfu-db/connector-x/issues/386")
+    def test_read_sql_connectorx(self):
```

Comment:
Should this be a separate test or a subtest under `test_read_sql`?  It seems like it is a subtest.

Reply:
It it made a separate test because we need to use ```@unittest.skipIf```